content/_data/changelogs/weekly.yml

```diff
@@ -14565,6 +14565,74 @@
   # pull: 6313 (PR title: Clarify that the changelog entry should be in the imperative mood)
   # pull: 6314 (PR title: Bump script-security from 1138.v8e727069a_025 to 1140.vf967fb_efa_55a_)
 
+  - version: '2.338'
+    date: 2022-03-06
+    changes:
+      - type: rfe
+        category: developer
+        pull: 6317
+        authors:
+          - glasswalk3r
+          - 'TODO: alceu.freitas (alceu.freitas@semantix.com.br)'
+        pr_title: Refactor `translation-tool.pl`
+        message: |-
+          Developer: Add a <code>--debug</code> option to the translation tool to help in troubleshooting.
+           Developer: Improve the readability of the translation tool command-line interface (CLI).
```

Comment:
Question: Do we need to add the prefix "Developer:" for each sentence in this entry?

Reply:
Pattern in other changelog entries has been to have at most one "Developer:" per changelog entry.